diff -Naur orig/designer/designer.pro qwt-6.1.2/designer/designer.pro
--- orig/designer/designer.pro	2016-03-14 12:45:12.215721714 +0000
+++ qwt-6.1.2/designer/designer.pro	2016-03-14 12:45:47.368367347 +0000
@@ -42,7 +42,7 @@
 
 
     TEMPLATE        = lib
-    TARGET          = qwt_designer_plugin
+    TARGET          = qwt_designer_plugin-qt5
 
     DESTDIR         = plugins/designer
 
@@ -84,7 +84,7 @@
         # into the plugin. Not supported on Windows !
 
         QMAKE_RPATHDIR *= $${QWT_INSTALL_LIBS}
-        qwtAddLibrary($${QWT_OUT_ROOT}/lib, qwt)
+        qwtAddLibrary($${QWT_OUT_ROOT}/lib, qwt-qt5)
 
         contains(QWT_CONFIG, QwtDll) {
 
diff -Naur orig/src/src.pro qwt-6.1.2/src/src.pro
--- orig/src/src.pro	2016-03-14 12:45:12.199721421 +0000
+++ qwt-6.1.2/src/src.pro	2016-03-14 12:45:26.223979002 +0000
@@ -17,7 +17,7 @@
 QWT_OUT_ROOT = $${OUT_PWD}/..
 
 TEMPLATE          = lib
-TARGET            = $$qwtLibraryTarget(qwt)
+TARGET            = $$qwtLibraryTarget(qwt-qt5)
 
 DESTDIR           = $${QWT_OUT_ROOT}/lib
 
